src/components/sessions/activeSession/confusion/ConfusionBarometer.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Checkbox, Header } from 'semantic-ui-react'
+import { FormattedMessage } from 'react-intl'
+
+import withCSS from '../../../../lib/withCSS'
+
+const ConfusionBarometer = ({ data }) => {
+  const calculateAverage = (allData, category) => {
+    const values = []
+    switch (category) {
+      case 'difficulty':
+        allData.map(({ difficulty }) => (values.push(difficulty)))
+        break
+      case 'comprehensibility':
+        allData.map(({ comprehensibility }) => (values.push(comprehensibility)))
+        break
+      default:
+        break
+    }
+    const sum = values.reduce((previous, current) => previous + current)
+    return sum / values.length
+  }
+
+  return (
+    <div>
+      <Header dividing as="h2" content="Confusion-Barometer" />
+      {/* TODO semantic-ui styling import */}
+      <Checkbox toggle label="Aktiviert" />
+      <div className="difficulty">
+        <p className="sectionTitle">
+          <FormattedMessage
+            defaultMessage="Difficulty"
+            id="pages.runningSession.confusionBarometer.paragraph.difficulty" // TODO correct naming of identifier
```

Comment:
We probably shouldn't grow the ids that long. Need some conventions @rschlaefli.